scenes/abstract.py

```diff
@@ -1,19 +1,21 @@
 from abc import ABC
 from typing import Tuple
 
+import pymunk
 from pygame.event import Event
 from pygame.surface import Surface
 
 
 class AbstractScene(ABC):
     def __init__(self, display: Surface) -> None:
-        self.display = display
-
+        self.display: Surface = display
+        self.size_sc: tuple = display.get_size()
+        self.space: pymunk.Space() = pymunk.Space()
```

Comment:
I do not this the space should be the field of the `AbstractScene`. The scene inherited from the `AbstractScene` may not use pymunk space at all. If you want, you can write another abstract class out of this one and name it `AbstractPymunkScene`.